tf_rl/simulation/karpathy_game.py

```diff
@@ -296,3 +297,10 @@ def to_html(self, stats=[]):
 
         return scene
 
+    def setup_draw(self):
```

Comment:
but the whole point of hasattr's above is that we do not need those functions? Possibly it is instructive to keep them there, so that people know this is an option, but then at least add a comment that that's their purpose and that in this particular case you can remove them and the simulator will continue working.